sse2neon.h

```diff
@@ -2650,10 +2701,7 @@ FORCE_INLINE __m128i _mm_unpackhi_epi64(__m128i a, __m128i b)
 // shift to right
 // https://msdn.microsoft.com/en-us/library/bb514041(v=vs.120).aspx
 // http://blog.csdn.net/hemmingway/article/details/44828303
-FORCE_INLINE __m128i _mm_alignr_epi8(__m128i a, __m128i b, const int c)
-{
-    return (__m128i) vextq_s8((int8x16_t) a, (int8x16_t) b, c);
-}
+#define _mm_alignr_epi8(a, b, c) ((__m128i) vextq_s8((int8x16_t) (b), (int8x16_t) (a), (c)))
```

Comment:
Why was the original inlined function replaced with macro? It might sacrifice the capability to check parameter types by compilers.

Reply:
Clang requires `c` be an integer literal, as seen by #15.

Reply:
Thanks for pointing this. Can you add corresponding comments in file `sse2neon.h`?

Reply:
Done